src/form/editors/base/MarionetteEditorPrototype.js

```diff
@@ -281,7 +285,7 @@ export default {
              */
             commit: function (options) {
                 options = options || {};
-                var error = this.validate();
+                var error = this.validate(true);

```

Comment:
I suggest that `commit()` must be treated equally to external call of `validate()`.
So no `internal` parameter is needed, the behavior is the same.